lib/handlers/quote/quote.ts

```diff
@@ -42,6 +42,47 @@ export class QuoteHandler extends APIGLambdaHandler<
 > {
   public async handleRequest(
     params: HandleRequestParams<ContainerInjected, RequestInjected<IRouter<any>>, void, QuoteQueryParams>
+  ): Promise<Response<QuoteResponse> | ErrorResponse> {
+    const { chainId, metric } = params.requestInjected
+    const startTime = Date.now()
+
+    let result: Response<QuoteResponse> | ErrorResponse
+
+    try {
+      result = await this.handleRequestInternal(params)
+
+      switch (result.statusCode) {
+        case 200:
```

Comment:
shall we just check for inequalities rather than specific error codes?

Reply:
precisely, something like that. or mayb 
if >=200 && < 300: whatever
if >=400 && < 500: lalalal

i think you get the message

Reply:
I think right now we can more easily extend if we want to have more granularity, but I dont have a strong opinion either way. I'm thinking I would merge it given the 2 approvals and then we can iterate if you feel it would be better that way